### DISCUSSION TITLE
using a formula in slider VOUT command

### DISCUSSION THREAD

**Discussion by bravowren** [artisan-roaster-scope/artisan#621] [Q&A] [answered]
I'm trying to send a VOUT command _**out(ch,v)**_, but with a 0-5V output inverse to the Event value, where **v (voltage out) = 5-E (Event Value).** 
E.G. Event Value of 0 sends a voltage out command of 5, and Event Value of 5 sends a voltage out command of 0. 
This is easily done with buttons, but I need to use sliders/quantifiers. 
In slider config, I tried the VOUT command _**out(0,5-{})**_, which did not work. "Factor" and "Offset" don't help me here. Any suggestions?

In case anyone is wondering what my application is, I need to control a 2-wire DC motor, which can not be driven with PWM. The voltage controlled dc-dc regulator board I'm using as an interface (between Phidgets OUT1000 and the device) has 0-5V control input inversely proportional to the output. If this does not work, I'm going to have to figure out how to filter PWM output of a DCC1002, in order to get to acceptable ripple for the motor.

**Comment by MAKOMO** [comment, marked as answer]
Works for me using the current Artisan v2.4.4. Set the slider to min=0, max=5 and use VOUT Command action with Command "out(0,5-{})" (offset=0, factor=1). Put the slider to 5 and the output is 0V, put the slider to 0 and the output is 5V. The command processing of sliders is the same as for buttons. Same code is used.

<img width="1060" alt="Screenshot 2021-06-03 at 09 57 18" src="https://user-images.githubusercontent.com/13909011/120609128-34f89200-c452-11eb-8f1b-7a09b28812eb.png">

**Reply by bravowren** [reply]
Thanks Marko. Yes it works. I had messed up by assuming the factor applies to the formula "x-{}", but figured out it only applies to the "{}" portion.
